src/components/ExportButton.tsx

```diff
@@ -177,13 +180,13 @@ export const ExportButton = ({ children, className }: ExportButtonProps) => {
                       <option value="standard">standard</option>
                     </>
                   )}
-                  {type === 'ply' && (
+                  {type === 'stl' && (
                     <>
                       <option value="ascii">ascii</option>
                       <option value="binary">binary</option>
                     </>
                   )}
-                  {type === 'stl' && (
+                  {type === 'ply' && (
                     <>
                       <option value="ascii">ascii</option>
                       <option value="binary_little_endian">
```

Comment:
This looked like an existing bug where we were giving wrong options for stl and ply